monitor.py

```diff
@@ -476,16 +476,27 @@ def get_nightly(build_log_url,build_log_response, job_platform):
         else:
             nightly = job_platform+"-latest-"+ nightly_log_match.group(2)
     else:
+        nightly=""
```

Comment:
Do we need this initialisation here. If not can we remove this.

Reply:
done